src/sfizz/Region.cpp

```diff
@@ -579,13 +579,30 @@ bool sfz::Region::parseOpcode(const Opcode& rawOpcode, bool cleanOpcode)
             filters[filterIndex].resonance = opcode.read(Default::filterResonance);
         }
         break;
-    case_any_ccN("cutoff&"): // also cutoff_oncc&, cutoff_cc&, cutoff&_cc&
+    case hash("cutoff&_cc&"): case hash("cutoff&_oncc&"):
```

Comment:
What was the issue with the previous switch?

Reply:
Maybe I misread the macro definition for cas_any_ccN, but my recollection is that the way I edited it means it won't get confused with "smoothcc" or "stepcc", which, as you probably know, is still unimplemented at this time.